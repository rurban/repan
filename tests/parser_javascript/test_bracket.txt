# Testing bracket types.

/(ab)(?:cd)(?=ef)(?!gh)(?<=ij)(?<!kl)/J
    print

/(?|ab)/J
    print

/(?<name>ab)cd\k<name>/J
    print
