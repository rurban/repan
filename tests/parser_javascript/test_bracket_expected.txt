# Testing bracket types.

/(ab)(?:cd)(?=ef)(?!gh)(?<=ij)(?<!kl)/J
    print
    /(ab)(?:cd)(?=ef)(?!gh)(?<=ij)(?<!kl)/

/(?|ab)/J
    print
    Pattern parse failed: Invalid subpattern type
        /[ HERE ]>>(?|ab)/

/(?<name>ab)cd\k<name>/J
    print
    /(?<name>ab)cd\g{name}/
